app/src/controllers/plugins/selectors.js

```diff
@@ -39,12 +39,15 @@ export const pluginsSelector = (state) => domainSelector(state).plugins;
 export const pluginByNameSelector = (state, name) =>
   pluginsSelector(state).find((plugin) => plugin.name === name);
 
-const globalIntegrationsSelector = (state) =>
+export const globalIntegrationsSelector = (state) =>
   domainSelector(state).integrations.globalIntegrations || [];
 const projectIntegrationsSelector = (state) =>
   domainSelector(state).integrations.projectIntegrations || [];
 
 export const availablePluginsSelector = createSelector(pluginsSelector, filterAvailablePlugins);
+export const availablePluginNamesSelector = createSelector(availablePluginsSelector, (plugins) =>
```

Comment:
`availablePluginsSelector` used for project settings page to hide disabled plugins and plugins with admin access only.
So if you want to get all plugin names, please, use `pluginsSelector`.

Reply:
done.
Changed to filtering only enabled plugins